wpilibj/src/main/java/edu/wpi/first/wpilibj/Telemetry.java

```diff
@@ -0,0 +1,240 @@
+
+  private Telemetry() {
+    throw new UnsupportedOperationException("This is a utility class!");
+  }
+
+  /**
+   * Gets a child telemetry table.
+   * @param name table name
+   * @return table
+   */
+  public static TelemetryTable getTable(String name) {
+    return m_root.getTable(name);
+  }
+
+  /**
+   * Indicates duplicate values should be preserved. Normally duplicate values are ignored.
+   *
+   * @param name the name
+   */
+  public static void keepDuplicates(String name) {
+    m_root.keepDuplicates(name);
+  }
+
+  /**
+   * Sets property for a value. Properties are stored as a key/value map.
+   *
+   * @param name the name
+   * @param key property key
+   * @param value property value
+   */
+  public static void setProperty(String name, String key, String value) {
+    m_root.setProperty(name, key, value);
+  }
+
+  /**
+   * Sets custom data type for a value. Generally not necessary.
+   *
+   * @param name the name
+   * @param typeString type string
+   */
+  public static void setTypeString(String name, String typeString) {
+    m_root.setTypeString(name, typeString);
+  }
+
+  /**
+   * Logs a generic object.
+   *
+   * @param name the name
+   * @param value the value
+   */
+  public static void log(String name, Object value) {
+    m_root.log(name, value);
+  }
+
+  /**
+   * Logs a generic array.
+   *
+   * @param name the name
+   * @param value the value
+   */
+  public static void log(String name, Object[] value) {
+    m_root.log(name, value);
+  }
+
+  /**
+   * Logs a boolean.
+   *
+   * @param name the name
+   * @param value the value
+   */
+  public static void log(String name, boolean value) {
+    m_root.log(name, value);
+  }
+
+  /**
+   * Logs a byte.
+   *
+   * @param name the name
+   * @param value the value
+   */
+  public static void log(String name, byte value) {
+    m_root.log(name, value);
+  }
+
+  /**
+   * Logs a short.
+   *
+   * @param name the name
+   * @param value the value
+   */
+  public static void log(String name, short value) {
+    m_root.log(name, value);
+  }
+
+  /**
+   * Logs an int.
+   *
+   * @param name the name
+   * @param value the value
+   */
+  public static void log(String name, int value) {
+    m_root.log(name, value);
+  }
+
+  /**
+   * Logs a long.
+   *
+   * @param name the name
+   * @param value the value
+   */
+  public static void log(String name, long value) {
+    m_root.log(name, value);
+  }
+
+  /**
+   * Logs a float.
+   *
+   * @param name the name
+   * @param value the value
+   */
+  public static void log(String name, float value) {
+    m_root.log(name, value);
+  }
+
+  /**
+   * Logs a double.
+   *
+   * @param name the name
+   * @param value the value
+   */
+  public static void log(String name, double value) {
+    m_root.log(name, value);
+  }
+
+  /**
+   * Logs a String.
+   *
+   * @param name the name
+   * @param value the value
+   */
+  public static void log(String name, String value) {
```

Comment:
Shouldn't be needed since String inherits from Object

Reply:
This is common enough that we want to fast-path it (not go through all the if checks in the Object path).